framework/src/play/utils/Os.java

```diff
@@ -0,0 +1,27 @@
+package play.utils;
+
+/**
+ * Os detections
+ */
+public class Os {
```

Comment:
What about renaming `Os` to `OS`?
It seems more natural for me because "OS" is a well-known acronym.